IdentityCore/src/controllers/MSIDLocalInteractiveController.m

```diff
@@ -94,9 +104,33 @@ - (void)acquireTokenImpl:(nonnull MSIDRequestCompletionBlock)completionBlock
     }];
 }
 
-- (BOOL)completeAcquireToken:(nonnull NSURL *)resultURL
+- (void)promptBrokerInstallWithResponse:(MSIDWebMSAuthResponse *)response completionBlock:(MSIDRequestCompletionBlock)completion
 {
-    return NO;
+#if TARGET_OS_IPHONE
+    if ([NSString msidIsStringNilOrBlank:response.appInstallLink])
+    {
+        NSError *appInstallError = MSIDCreateError(MSIDErrorDomain, MSIDErrorInternal, @"App install link is missing. Incorrect URL returned from server", nil, nil, nil, self.requestParameters.correlationId, nil);
+        completion(nil, appInstallError);
+        return;
+    }
+
+    NSError *brokerError = nil;
+    MSIDBrokerController *brokerController = [[MSIDBrokerController alloc] initWithInteractiveRequestParameters:self.interactiveRequestParamaters
+                                                                                           tokenRequestProvider:self.tokenRequestProvider
+                                                                                              brokerInstallLink:[NSURL URLWithString:response.appInstallLink]
+                                                                                                          error:&brokerError];
+
+    if (!brokerController)
+    {
+        completion(nil, brokerError);
```

Comment:
Should we check for nil completion block? it still possible to pass nil despite nonnull annotation, it would be a rare case though.

Reply:
Added checks in controllers